src/main/java/org/openrewrite/java/logging/PrintStackTraceToLogError.java

```diff
@@ -78,23 +84,34 @@ public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, Execu
                     J.ClassDeclaration clazz = getCursor().firstEnclosingOrThrow(J.ClassDeclaration.class);
                     Set<J.VariableDeclarations> loggers = FindFieldsOfType.find(clazz, framework.getLoggerType());
                     if (!loggers.isEmpty()) {
-                        m = framework.getErrorTemplate(this, "\"Exception\"")
-                                .apply(
-                                        new Cursor(getCursor().getParent(), m),
-                                        m.getCoordinates().replace(),
-                                        loggers.iterator().next().getVariables().get(0).getName(),
-                                        m.getSelect());
-                        if (framework == LoggingFramework.JUL) {
-                            maybeAddImport("java.util.logging.Level");
-                        }
+                        J.Identifier logField = loggers.iterator().next().getVariables().get(0).getName();
+                        m = replaceMethodInvocation(m, logField);
+                    } else if (clazz.getAllAnnotations().stream().anyMatch(lombokLogAnnotationMatcher::matches)) {
+                        String fieldName = loggerName == null ? "log" : loggerName;
```

Comment:
By default fields are named `log`, as per: https://projectlombok.org/features/log

Didn't quite feel like parsing any lombok configuration files in case folks set a different `lombok.log.fieldName`.
If they do that, then can set the `loggerName` option to have that be used for the logging statements instead.